src/melee.cpp

```diff
@@ -322,6 +322,13 @@ float Character::hit_roll() const
         hit -= 2.0f;
     }
 
+    // Difficult to land a hit while prone
+    if( is_on_ground() ) {
+        hit -= one_in( 4 ) ? 4.0f : 10.0f;
+    } else if( is_crouching() ) {
+        hit -= one_in( 4 ) ? 0.0f : 4.0f;
+    }
```

Comment:
I don't like injecting randomness in a bunch of different places as it makes the final outcome difficult to predict, make this a flat penalty.

Reply:
I changed these to flat values (8.0f and 2.0f respectively).